### PR TITLE
feat(draw/sw): added support for 3 bpp font rendering

### PR DESCRIPTION
**Pull request by RomanLuchyshyn**
There was support of rendering for 3 bpp font rendering. But in commit 8f6b1f8 this functionality was  accidentally  removed in context of issue #4059 .
This code add this back.

### CODE REVIEW

**Comment by kisvegabor** [comment]
How can you render A3 bitmaps? I don't thing that SW render supports it now.

**Comment by RomanLuchyshyn** [comment]
> How can you render A3 bitmaps? I don't thing that SW render supports it now.

Yes, it renders good. I have already tested on my hw and SDL Linux. I didn't change other renderers because I don't hardware for those ones.

**Comment by kisvegabor** [comment]
Got it now. I've continued your PR and updated some tests too. Please test if it works on your end too.

**Comment by RomanLuchyshyn** [comment]
> Got it now. I've continued your PR and updated some tests too. Please test if it works on your end too.

I've just tested on my env. It's fine. 
Thank you very much.

**Review by C47D — APPROVED** [review]
LGTM

File: src/font/lv_font.h [hunk 1]
```diff
@@ -39,6 +39,7 @@ typedef enum {
     /**< Legacy simple formats with no byte padding at end of the lines*/
     LV_FONT_GLYPH_FORMAT_A1     = 0x01, /**< 1 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A2     = 0x02, /**< 2 bit per pixel*/
+    LV_FONT_GLYPH_FORMAT_A3     = 0x03, /**< 3 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A4     = 0x04, /**< 4 bit per pixel*/
     LV_FONT_GLYPH_FORMAT_A8     = 0x08, /**< 8 bit per pixel*/
```
Comment (liamHowatt) [inline src/font/lv_font.h:44]: @kisvegabor these don't get OR'd together anywhere, do they?

**Review by liamHowatt — COMMENTED** [review]
It might be good to mention 3 bpp in the docs and say that only compressed 3 bpp is supported.

https://github.com/lvgl/lvgl/blob/master/docs/details/main-components/font.rst
https://docs.lvgl.io/master/details/main-components/font.html

This is the only mention of 3 bpp

> - `LV_FONT_MONTSERRAT_28_COMPRESSED`: Same as normal 28 px font but stored as a Compressed fonts with 3 bpp

File: src/font/lv_font.h (hunk 1 above)
Comment (kisvegabor) [reply src/font/lv_font.h:44]: Correct, it's not OR-ed.

**Review by liamHowatt — APPROVED** [review]
Thank you both!

**Review by uLipe — APPROVED** [review]

**Comment by RomanLuchyshyn** [comment]
Thank you very much, guys!

**Pull request merged**